ffi/nodejs/rgb_node.js

```diff
@@ -0,0 +1,30 @@
+const lib = require('./build/Release/rgb_node')
+
+function array_to_pointer(js_array) {
```

Comment:
I am pretty sure there should be a way in JS to get a pointer to the heap where the array is stored w/o the need for allocation & copy at all

Reply:
from the documentation I read this is not possible. but if you have an example of code doing that, I'm happy to change this. I don't appreciate the solution either, actually I was surprised to learn that swig doesn't handle arrays automatically, seems a pretty useful feature to me